index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link href="style.css" rel="stylesheet">
+    <script src="index.js" defer></script>
+    <title>To-Do List</title>
+</head>
+<body>
+   <div class="container">
+    <div class="input-sec">
+        <p class="black heading">Today's To Do</p> 
```

Comment:
- [ ] Kindly, use heading tags since they are semantically correct for your website and accessibility. For example, you can use `h1`.